tests/backup/backup_upgrade_test.go

```diff
@@ -1341,11 +1341,11 @@ var _ = Describe("{PXBackupClusterUpgradeTest}", Label(TestCaseLabelsMap[PXBacku
 					err = Inst().V.RefreshDriverEndpoints()
 					log.FailOnError(err, "Refresh Driver Endpoints failed")
 
-					urlToParse := fmt.Sprintf("%s/%s", Inst().StorageDriverUpgradeEndpointURL, Inst().StorageDriverUpgradeEndpointVersion)
-					u, err := url.Parse(urlToParse)
-					log.FailOnError(err, fmt.Sprintf("error parsing PX version the url [%s]", urlToParse))
-					err = Inst().V.ValidateDriver(u.String(), true)
-					dash.VerifyFatal(err, nil, fmt.Sprintf("verify volume driver after upgrade to %s", version))
+					storageNodes := node.GetWorkerNodes()
+					for index := range storageNodes {
+						err = Inst().V.WaitDriverUpOnNode(storageNodes[index], time.Minute*5)
+						dash.VerifyFatal(err, nil, "Validate if PX is up")
```

Comment:
Include node name in the description? storageNodes[index].Name

Reply:
Okay doesn't harm.